### DISCUSSION TITLE
Automatic change profiles base on condition?

### DISCUSSION THREAD

**Discussion by MercTK** [Rem0o/FanControl.Releases#2746] [Ideas]
Like a smart home feature but adapt to this app. For example, if I open a game (full screen app) the fan profile will change or schedule a profiles.

**Comment by willygeerts** [comment]
I had the same idea, even add the possibility to use time based profile changing.
For example, at night my backups are running, pulling high cpu for long time, I'd like to plan/schedule my profile "high cooling".

The condition based changing profile is a very nice thought, combining it with an existing if-this-then-that can be an option so as not to have to program all the detection logic into FanControl.

**Reply by Rem0o** [reply]
FanControl has a basic CLI interface, which allows to change configuration with it. See the docs (https://getfancontrol.com/docs/)

**Reply by willygeerts** [reply]
CLI? great!
My bad I never read up on it, thanks Rem0o for the tip.
